src/SFA.DAS.Reservations.Application.UnitTests/Reservations/Commands/CreateReservationLevyEmployer/WhenValidatingCreateReservationLevyEmployerCommand.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Castle.Core.Internal;
+using NUnit.Framework;
+using SFA.DAS.Reservations.Application.Reservations.Commands.CreateReservationLevyEmployer;
+
+namespace SFA.DAS.Reservations.Application.UnitTests.Reservations.Commands.CreateReservationLevyEmployer
+{
+    [TestFixture]
+    public class WhenValidatingCreateReservationLevyEmployerCommand
+    {
+        [TestCase(Int64.MinValue, Int64.MinValue)]
```

Comment:
No sure we need to test this boundaries, another test that should be done is to make sure that if one of the two required values is valid and the other is not then the result is not valid